slam-libraries/viam-cartographer/src/io/file_handler_test.cc

```diff
@@ -35,71 +37,58 @@ BOOST_AUTO_TEST_CASE(MakeFilenameWithTimestamp_success) {
 
 BOOST_AUTO_TEST_CASE(ListSortedFilesInDirectory_success) {
```

Comment:
[minor] can we add a test for if files were added in wrong order to ensure sorting of them is done by SLAM's `ListSortedFilesInDirectory`

Reply:
Yes, that's a good idea. Changing this test to reflect that.